scout/server/blueprints/variants/controllers.py

```diff
@@ -549,6 +549,9 @@ def update_variant_genes(store, variant_obj, genome_build):
             has_changed = True
             variant_gene = {"hgnc_id": hgnc_id}
             variant_obj["genes"].append(variant_gene)
+        variant_genes = variant_obj.get(
+            "genes"
+        )  # we need update variant_genes here, to get all fixes in one call to the function
```

Comment:
And here is the slightly more aggressive fix-them-in-the-same-go. No reason not to.